### PR TITLE
Add content/pt-br/containerization.md

### PR DESCRIPTION
**Pull request by edsoncelio**
Add containerization.md localization to ptbr.

### CODE REVIEW

File: content/pt-br/containerization.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+title: Contêinerização
+status: Completed
+category: tecnologia
+---
+
+## O que é
+A conteinerização é o processo de agrupar uma aplicação e suas dependências em uma [imagem de contêiner](/container-image/). O processo de criação do contêiner requer adesão ao padrão [Open Container Initiative](https://opencontainers.org) (OCI). Desde que a saída seja uma imagem de contêiner que atenda a esse padrão, qual a ferramenta de conteinerização usada não importa.
```
Comment (jessicalins) [inline content/pt-br/containerization.md:8]:
```suggestion
A conteinerização é o processo de agrupar uma aplicação e suas dependências em uma [imagem de contêiner](/container-image/). O processo de criação do contêiner requer adesão ao padrão [Open Container Initiative](https://opencontainers.org) (OCI). Desde que a saída seja uma imagem de contêiner que atenda a esse padrão, a ferramenta de conteinerização usada não importa.
```

I think we can drop `qual` - what do you think?

File: content/pt-br/containerization.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+## Como isso ajuda
+As imagens de contêiner são leves (ao contrário das VMs tradicionais) e o processo de conteinerização requer apenas um arquivo com uma lista de dependências. Esse arquivo pode ser versionado e o processo de compilação automatizado, permitindo que uma organização se concentre em outras prioridades enquanto os processos automatizados cuidam da compilação. Uma imagem de contêiner é armazenada por um identificador exclusivo vinculado ao seu conteúdo e configuração exatos. À medida que os contêineres são programados e reprogramados, eles sempre são redefinidos para seu estado inicial, o que elimina inconsistências de configuração.
```
Comment (jessicalins) [inline content/pt-br/containerization.md:14]:
```suggestion
As imagens de contêiner são leves (ao contrário das VMs tradicionais) e o processo de conteinerização requer apenas um arquivo com uma lista de dependências. Esse arquivo pode ser versionado e o processo de compilação pode ser automatizado, permitindo que uma organização se concentre em outras prioridades enquanto os processos automatizados cuidam da compilação. Uma imagem de contêiner é armazenada por um identificador exclusivo vinculado ao seu conteúdo e configuração exatos. À medida que os contêineres são programados e reprogramados, eles sempre são redefinidos para seu estado inicial, o que elimina inconsistências de configuração.
```

File: content/pt-br/containerization.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+title: Contêinerização
+status: Completed
+category: tecnologia
+---
+
+## O que é
+A conteinerização é o processo de agrupar uma aplicação e suas dependências em uma [imagem de contêiner](/container-image/). O processo de criação do contêiner requer adesão ao padrão [Open Container Initiative](https://opencontainers.org) (OCI). Desde que a saída seja uma imagem de contêiner que atenda a esse padrão, qual a ferramenta de conteinerização usada não importa.
+
+## Problema relacionado
+Antes que o uso de contêineres se tornasse mais comum , as organizações usavam máquinas virtuais (VMs) para orquestrar várias aplicações em uma única [máquina baremetal](/bare_metal_machine/). As VMs são significativamente maiores que os contêineres e exigem um *hypervisor* para serem executados. Devido ao armazenamento, backup e transferência desses *templates* de VM maiores, a criação dos *templates* de VM também é lenta. Além disso, as VMs podem sofrer inconsistências nas configurações, o que viola o princípio da [imutabilidade](/immutable_infrastructure/).
```
Comment (jessicalins) [inline content/pt-br/containerization.md:11]:
```suggestion
Antes que o uso de contêineres se tornasse mais comum , as organizações usavam máquinas virtuais (VMs) para orquestrar várias aplicações em uma única [máquina bare metal](/bare_metal_machine/). As VMs são significativamente maiores que os contêineres e exigem um *hypervisor* para serem executados. Devido ao armazenamento, backup e transferência desses *templates* de VM maiores, a criação dos *templates* de VM também é lenta. Além disso, as VMs podem sofrer inconsistências nas configurações, o que viola o princípio da [imutabilidade](/immutable_infrastructure/).
```

When searching I've seen baremetal machine being translated to `máquina bare metal` in most of the places - maybe we could go with this translation?

**Review by jessicalins — COMMENTED** [review]
looks good! just a few suggestions

**Comment by edsoncelio** [comment]
Thanks @jessicalins all of your suggestions are applied :D

**Review by jessicalins — APPROVED** [review]
LGTM! :sparkler:

**Pull request merged**